internal/stack/stacks_test.go

```diff
@@ -263,6 +263,23 @@ func TestParseStack(t *testing.T) {
 				"example.com/foo/bar.baz",
 			},
 		},
+		{
+			name: "elided frames",
+			give: joinLines(
+				"goroutine 1 [running]:",
+				"example.com/foo/bar.baz()",
+				"	example.com/foo/bar.go:123",
+				"...3 frames elided...",
```

Comment:
In addition to this fixed data test case, can we add or update a test that creates a goroutine with a deep stack that leads to elided frames, and verify that? That way, we'll have coverage that will detect if the elided message has changed.

This is a real concern, when the eliding logic was [originally added](https://go-review.googlesource.com/c/go/+/70914/1/src/runtime/traceback.go#723), it used "stack frames omitted". Given that it has changed, we should have a test that will detect changes by relying on the real runtime logic.